databases/mysql-ob/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+appVersion: "5.7"
+description: A Helm chart for WSO2 Open Banking Datasources
+name: mysql-ob
+version: 1.4.0-0
```

Comment:
@kdanW this has to be changed to `1.4.0-1`. This value has to map with the Helm resource Git release tag number.